packages/_debian-common/changelog

```diff
@@ -1,8 +1,9 @@
 libsuperiormysqlpp (0.5.1) UNRELEASED; urgency=medium
 
-  * 
+  [ Adam Stepan ]
+  * Fix compilation with libc++
```

Comment:
Please note what actually changed, fix of clang compilation is essentially a sideeffect.